spec/support/features/session_helper.rb

```diff
@@ -35,6 +35,16 @@ def sign_up
       user
     end
 
+    def sign_up_with_sp(loa: 'loa1')
+      allow(FeatureManagement).to receive(:prefill_otp_codes?).and_return(true)
+      user = sign_up_and_set_password
+      fill_in 'Phone', with: '202-555-1212'
+      send("#{loa}_sp_session")
+      select_sms_delivery
+
+      user
+    end
```

Comment:
It's not complaining about that offense for me in my proposed method above. What it is complaining about is that `loa3` is an uncommunicative parameter name, but I think we can ignore that offense.

The advantage of my method is it makes the test faster because it skips the `sign_up_and_set_password` part.